pkgs/applications/misc/pomodoro/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, fetchFromGitHub, rustPlatform }:
```

Comment:
```suggestion
{ lib, fetchFromGitHub, rustPlatform, Foundation }:
```

Reply:
this also requires a `buildInputs = lib.optionals stdenv.hostPlatform.isDarwin [ Foundation ];` in the derivation